setup.cfg

```diff
@@ -0,0 +1,18 @@
+[flake8]
+show-source = True
+ignore = E203, E731, W503
+max-line-length = 88
+exclude = boostrap.py
+
+[black]
+exclude = bootstrap.py
```

Comment:
W Reg I put the black configuration into pyproject.toml.
If you want to put the same configuration here, please take a look.

Reply:
Moved